### PR TITLE
(chore) removed metrics on the header

### PR DESCRIPTION
**Pull request by its-kios09**
## Requirements

- [ ] This PR has a title that briefly describes the work done, including the ticket number if there is a ticket.
- [ ] My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide).
- [ ] I checked for feature overlap with [**existing widgets**](https://om.rs/directory).

## Screenshots
caseMana

*None.*

## Related Issue

*None.*

## Other

*None.*

### CODE REVIEW

File: packages/esm-patient-clinical-view-app/src/case-management/metrics/case-management-header.component.tsx [hunk 1]
```diff
@@ -3,26 +3,27 @@ import { useTranslation } from 'react-i18next';
...
+  const metricsTitle = t(' ', 'Case Manager');
   const handleAddCase = () => {
     launchWorkspace('case-management-form', {
       workspaceTitle: 'Case Management Form',
     });
   };
   return (
     <div className={styles.metricsContainer}>
-      <span className={styles.metricsTitle}>{metricsTitle}</span>
+      <span className={styles.metricsTitle}> </span>
```
Comment (donaldkibet) [inline packages/esm-patient-clinical-view-app/src/case-management/metrics/case-management-header.component.tsx:19]:
```suggestion
    
```

File: packages/esm-patient-clinical-view-app/src/case-management/metrics/case-management-header.component.tsx [hunk 2]
```diff
@@ -3,26 +3,27 @@ import { useTranslation } from 'react-i18next';
...
 import { Button } from '@carbon/react';
 import styles from './case-management-header.scss';
-import { launchWorkspace, navigate } from '@openmrs/esm-framework';
+import { launchWorkspace, navigate, useSession } from '@openmrs/esm-framework';
 
 const MetricsHeader = () => {
   const { t } = useTranslation();
-  const metricsTitle = t('caseManager', 'Case Manager summaries');
+  const { user } = useSession();
+  const metricsTitle = t(' ', 'Case Manager');
```
Comment (donaldkibet) [inline packages/esm-patient-clinical-view-app/src/case-management/metrics/case-management-header.component.tsx:11]:
```suggestion
  const metricsTitle = t('caseManager', 'Case Manager');
```

**Review by donaldkibet — APPROVED** [review]
Thanks @its-kios09 feel free to merge once you address suggestions

File: packages/esm-patient-clinical-view-app/src/case-management/table/case-management-list-active-component.tsx [hunk 3]
```diff
@@ -83,9 +79,11 @@ const CaseManagementListActive: React.FC<{ setActiveCasesCount: (count: number)
       ),
     }));
 
+  // Update the count of active cases whenever `filteredCases` changes
   useEffect(() => {
-    setActiveCasesCount(filteredCases?.length || 0);
-  }, [filteredCases, setActiveCasesCount]);
+    const count = filteredCases?.length || 0;
+    setActiveCasesCountState(count);
+  }, [filteredCases]);
```
[the remark this answers is not in the document]
Comment (its-kios09) [reply packages/esm-patient-clinical-view-app/src/case-management/table/case-management-list-active-component.tsx]: I have moved it to tabs i don't if it is okay @donaldkibet

File: packages/esm-patient-clinical-view-app/src/case-management/table/case-management-list-inactive-component.tsx [hunk 4]
```diff
@@ -81,9 +79,11 @@ const CaseManagementListInActive: React.FC<{ setInactiveCasesCount: (count: numb
       ),
     }));
 
+  // Update the count of inactive cases whenever `filteredCases` changes
   useEffect(() => {
-    setInactiveCasesCount(filteredCases?.length || 0);
-  }, [filteredCases, setInactiveCasesCount]);
+    const count = filteredCases?.length || 0;
+    setInactiveCasesCountState(count);
+  }, [filteredCases]);
```
[the remark this answers is not in the document]
Comment (its-kios09) [reply packages/esm-patient-clinical-view-app/src/case-management/table/case-management-list-inactive-component.tsx]: same to here

File: packages/esm-patient-clinical-view-app/translations/en.json [hunk 5]
```diff
@@ -1,10 +1,10 @@
 {
+  " ": "Case Manager",
   "accidentOrTrauma": "Accident/Trauma",
   "actions": "Actions",
-  "activeCase": "Active cases",
   "activeCases": "Active cases",
   "add": "Add",
-  "addCase": "add a case",
+  "addCases": "Add a case",
```
[the remark this answers is not in the document]
Comment (its-kios09) [reply packages/esm-patient-clinical-view-app/translations/en.json]: I have changed it to Add case

[the remark this answers is not in the document]
Comment (ojwanganto) [reply packages/esm-patient-clinical-view-app/translations/en.json]: Thanks.

File: packages/esm-patient-clinical-view-app/src/case-management/table/case-management-list-inactive-component.tsx [hunk 6]
```diff
@@ -70,10 +67,8 @@ const CaseManagementListInActive: React.FC<{ setInactiveCasesCount: (count: numb
           {uppercaseText(extractNameString(caseData.personB.display))}
         </ConfigurableLink>
       ),
-      // reason: t('assignedTo', 'Assigned to ') + caseData.personB.display,
       dateofstart: new Date(caseData.startDate).toLocaleDateString(),
       dateofend: new Date(caseData.endDate).toLocaleDateString(),
-      // notes: '-',
       action: (
         <OverflowMenu flipped={document?.dir === 'rtl'} aria-label="overflow-menu">
           {/* <OverflowMenuItem itemText="Transfer Case" /> */}
```
Comment (donaldkibet) [inline packages/esm-patient-clinical-view-app/src/case-management/table/case-management-list-inactive-component.tsx:74]: We can remove commented code

**Review by donaldkibet — APPROVED** [review]
Thanks @its-kios09

**Pull request merged**